completions/bash/clr-installer

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# shellcheck disable=SC2207
+# -----------------------------------------------------------------------
+#   Clear Linux OS* Installer - autocompletion script
+#
+#   Author: Lucius Hu - http://github.com/lebensterben
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, version 2 or later of the License.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------
+
+_clr_installer() {
+  local IFS opts prev cur
+
+  _init_completion -s || return
+
+  case "$prev" in
+    --archive|--copy-network|--iso|--keep-image|--reboot|--stub-image|--swupd-skip-diskspace-check|--telemetry)
+      COMPREPLY=($(compgen -W "true false" -- "$cur"))
+      return
+      ;;
+    --block-device)
+      local blockdevice; blockdevice="$(lsblk --noheadings --raw --paths | cut -d' ' -f1 | tr '\n' ',')"
+      # The trick is to insert any UNICODE space(s), e.g. non-breaking space, before the message
+      #    so that 'word' is not likely to be matched by a user's input
+      # And add at least one other 'word' start with any UNICODE space(s)
+      #    otherwise completion system will push the only candidate to the command line
+      # The drawback is, this special character may not shown correctly, for example in `tty`, it's
+      #    it's shown as a rectangle
+
+      # here, before F and B, there is a UNICODE tag space U+E0020
+      opts="󠀠Format:\"alias1:filename1[,alias2:filename2,...]\" 󠀠BlockDevices:${blockdevice:0:-1}"
+      COMPREPLY=($(compgen -W "$opts" -- "$cur"))
+      return
+      ;;
+    --config)
```

Comment:
Is there any way to get this to expand with a path?
I would generally use --config <path>/[tab]

Same for other file expanders

Reply:
```bash
--config)
	COMPREPLY=($(compgen -f -X "!*.yaml" -- "$cur"))
	return
;;
```

I've replaced it with `_filedir yaml`, that's should be better.